src/compiler/scala/tools/nsc/transform/Fields.scala

```diff
@@ -613,7 +613,7 @@ abstract class Fields extends InfoTransform with ast.TreeDSL with TypingTransfor
           case _ => tree
         }
 
-      def initialized = Select(Ident(holderSym), initializedSym)
+      def initialized = Apply(Select(Ident(holderSym), initializedSym), Nil)
```

Comment:
Looks alright, and *now* seems to match the ScalaDoc on mkLazyLocalDef. Any chance of any other mismatch?

Reply:
Yes, but not that I can see on my phone.

I didn't even notice the comment the first time through.